examples/package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "start": "set NODE_ENV=development && webpack-dev-server --inline --hot --port=3000 --history-api-fallback",
     "build": "NODE_ENV=production webpack --define process.env.NODE_ENV='\"production\"' -p ",
-    "lint": "eslint src && find ./src/ -iname \"*.scss\" -exec sass-lint -v -q {} +"
+    "lint": "eslint src && find ./src/ -iname \"*.scss\" -exec sass-lint -v -q {} +",
```

Comment:
I'll let the `-v` flag because this is a standard one (--verbose)
But for the `-q` this is not standard at all (standard is for --quiet, here it means --no-exit).

So I'll fix that for the `-q`